sagemaker_processing/feature_transformation_with_sagemaker_processing_Dask/preprocess.py

```diff
@@ -0,0 +1,166 @@
+from __future__ import print_function
+from __future__ import unicode_literals
+import s3fs
+import dask.dataframe as dd
+import boto3
+import sys
+import time
+import json
+import time
+from dask.distributed import Client
+import argparse
+import os
+import warnings
+from tornado import gen
+
+import pandas as pd
+import numpy as np
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import StandardScaler, OneHotEncoder, LabelBinarizer, KBinsDiscretizer
+from sklearn.preprocessing import PolynomialFeatures
+from sklearn.compose import make_column_transformer
+
+from sklearn.exceptions import DataConversionWarning
+warnings.filterwarnings(action='ignore', category=DataConversionWarning)
+import joblib
+import dask.dataframe as dd
+s3_client = boto3.resource('s3')
+
+
+
+def get_resource_config():
+    resource_config_path = '/opt/ml/config/resourceconfig.json'
+    with open(resource_config_path, 'r') as f:
+        return json.load(f)
+
+def get_ip_from_host(host_name):
```

Comment:
https://pypi.org/project/retry/ makes the  code more readable. Is this retry really necessary?

Reply:
I don't think it is needed in this instance because the resource config file will get copied before the bootstap.py script kicks off